python/MetrologyData.py

```diff
@@ -202,6 +202,22 @@ def resids_boxplot(self, yrange=None, title=None):
         win.set_title(title)
         return win
 
+    # Below applies outlier clipping as in plot_statistics
+    def resids_clip_boxplot(self, nsigma=4, yrange=None, title=None):
+        win = plot.Window()
+	dz = self.resids
```

Comment:
I think you need to reconfigure your editor to avoid adding tabs.  PEP8 conventions set the single indentation level at 4 spaces, so this line should be indented 8 spaces.  The same applies to lines 284-287.